ui/components/CopyToCliboard.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from "react";
+import { useCallback, useState } from "react";
+import styled from "styled-components";
+import Icon, { IconType } from "./Icon";
+import Text from "./Text";
+
+const PointerIcon = styled(Icon)`
+  cursor: pointer;
+  svg {
+    fill: ${(props) => props.theme.colors.neutral30};
+  }
+  &.copied > svg {
+    fill: ${(props) => props.theme.colors.primary10};
+  }
+`;
+
+export default function CopyToClipboard({
```

Comment:
Would it make sense to replace the copy stuff in YamlView with this or is it different?